src/test/moment/calendar.js

```diff
@@ -5,13 +5,50 @@ import moment from '../../moment';
 
 module('calendar');
 
+var march1 = moment('2016-03-01');
+var march8 = moment('2016-03-08');
+var july1 = moment('2016-07-01');
+var jan1 = moment('2016-01-01');
+var dec28 = moment('2015-12-28');
+var nov1 = moment('2015-11-01');
+
 test('passing a function', function (assert) {
-    var a  = moment().hours(2).minutes(0).seconds(0);
-    assert.equal(moment(a).calendar(null, {
+    var day = moment().hours(2).minutes(0).seconds(0);
```

Comment:
2 am is the DST transition time in the United States. This test will break on the day of the transition. Use 12 instead.